src/components/SingleTweet.tsx

```diff
@@ -121,7 +118,14 @@ const SingleTweet: React.FC<Tweet> = ({
     <div>
       <Wrapper className="px-3">
         <Author className="my-2">
-          <Logo className="rounded-circle flex-shrink-0" src={userlogo} />
+          <Avatar
+            size="48"
+            round="50%"
+            twitterHandle="sitebase"
+            name={username}
+            src=""
+            // src={userlogo}
+          />
```

Comment:
not exactly what I meant, you need to render fallback avatar only in case when userlogo not exist, not in all cases. So uncomment both places where we using userlogo